src/transformers/models/deberta/tokenization_deberta.py

```diff
@@ -442,3 +442,7 @@ def _build_conversation_input_ids(self, conversation: "Conversation") -> List[in
         if len(input_ids) > self.model_max_length:
             input_ids = input_ids[-self.model_max_length :]
         return input_ids
+
+    @property
```

Comment:
If it's the same as Bloom_fast and for any other, missing the `Copied from` statements